site/docs/concepts/web-app.md

```diff
@@ -38,7 +41,7 @@ If you've never used Flow before, you'll be prompted to register before being is
 ## Navigating the web app
 
 When you log into the web app, you land on the **Welcome** page.
-There are four additional pages visible as tabs in the side navigation: **Captures**, **Collections**, **Materializations**, and **Admin**.
+There are four additional pages visible as tabs in the side navigation: **Sources (captures)**, **Collections**, **Destinations (materializations)**, and **Admin**.
```

Comment:
We've updated the left nav so throwing what entities each link represent in brackets.